django_celery_extensions/management/commands/run_celery.py

```diff
@@ -1,20 +1,34 @@
+import os
+
 import shlex
 import subprocess
 
 from django.conf import settings as django_settings
 from django.core.management.base import BaseCommand
-from django.utils.autoreload import run_with_reloader
+from django.utils.autoreload import run_with_reloader, DJANGO_AUTORELOAD_ENV
+
+import signal
+
+
+def kill_celery(celery_type, celery_settings):
+    kill_celery_command = f'pkill -9 -f "{get_celery_command(celery_type, celery_settings, add_queue=False)}"'
```

Comment:
there was bug. No one task was found and celery was not stopped. I must remove extra data a queue setting to find running command with pkill

Reply:
No there is nothing like this. And because django run this in subprocess which run subprocess in autoreload_celery there is no way how to do it :(